src/marqo/core/index_management/index_management.py

```diff
@@ -342,8 +353,7 @@ def _vespa_deployment_lock(self):
         else:
             try:
                 with self._zookeeper_deployment_lock:
-                    logger.info(f"Retrieved the distributed lock for index operations. ")
+                    logger.debug(f"Retrieved the distributed lock for index operations. ")
                     yield
             except ZookeeperLockNotAcquiredError:
-                raise OperationConflictError("Another index creation/deletion operation is in progress. "
-                                             "Your request is rejected. Please try again later")
+                raise OperationConflictError("Your indexes are being updated. Please try again shortly.")
```

Comment:
I feel like we need a docs link here? Overall this message is confusing. Could be a todo though

Reply:
will add a todo.